src/components/DateSelect/DateSelect.jsx

```diff
@@ -164,6 +171,7 @@ const DateSelect = createClass({
 			onSelectDate: _.noop,
 			isFontSizeRelative: false,
 			showCursorHighlight: true,
+			useSlidePanel: true,
```

Comment:
Do we want to make the default value be the more performant one?

Reply:
it would be considered a breaking change if we default it to false since it disables touch support.